enable_CAN_and_GPS.sh

```diff
@@ -21,8 +21,10 @@ if [[ $? != 0 ]]; then
   sudo ip link set dev vcan0 up
 fi
 
+sudo killall gpsd
 sudo service gpsd start
 sudo service gpsd stop
+sudo systemctl stop gpsd.socket
```

Comment:
Less important, but what's the goal of this section here? I don't entirely understand why we kill gpsd, then start it, then stop it, then stop its socket, etc. In general we should probably comment the startup script so it's clear what it's doing. 

Also, if gpsd has a systemd service then maybe we should configure it to auto-start via systemd when the Jetson boots up, thoughts?

Reply:
Added comments to code!

I'm a little less than enthusiastic about trying to figure out systemd right now but it might be interesting to figure that out at some point